src/recipes/sparkling-punch.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Sparkling Punch",
+    "description": "Sparkling apple cider, fresh orange juice, and lemonade concentrate blend into light and refreshing non-alcoholic punch in this easy recipe.",
+    "github": "kevinmel2000",
+    "ingredients": [
+        "1 banana",
+        "1 cup milk",
+        "1 teaspoon vanilla extract",
+		"1 egg",
+		"2 tablespoons white sugar",
+		"1 pinch ground cinnamon"
+    ],
```

Comment:
I think this is the wrong recipe entered.

Reply:
i push new fixed on new commit on this